Foundation/URLSession/URLSessionConfiguration.swift

```diff
@@ -156,7 +156,17 @@ open class URLSessionConfiguration : NSObject, NSCopying {
         return ephemeralConfiguration
     }
 
-    open class func background(withIdentifier identifier: String) -> URLSessionConfiguration { NSUnimplemented() }
+    open class func background(withIdentifier identifier: String) -> URLSessionConfiguration {
```

Comment:
We do not have a daemon handling background sessions outside of Darwin. This should be marked `@availability(*, unavailable)` with a message.

Reply:
@millenomi  I have reverted the implementation and added `@available(*, unavailable, message: "Not available on non-Darwin platforms")`